src/Workspaces/Core/Portable/Log/LogLevel.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.CodeAnalysis.Internal.Log
```

Comment:
This file should move to the shared project `SharedUtilitiesAndExtensions\Compilers`